dnf5daemon-client/commands/advisory/arguments.hpp

```diff
@@ -150,6 +150,8 @@ class AdvisorySeverityOption : public libdnf5::cli::session::AppendStringListOpt
               command,
               "advisory-severities",
               '\0',
+              /* Note for translators: "critical" etc. quoted words are
+                 literals that should not be translated. */
               _("Limit to packages in advisories with specified severity. List option. Can be "
```

Comment:
I am curios ehrther we should use `C_()` to add a context for translators?

Reply:
Translation contexts are for distinguishing two identical messages for a need of different translations. This is not the case. This message only exists at two places dnf5/commands/advisory_shared.hpp and dnf5daemon-client/commands/advisory/arguments.hpp. The messages has the same meaning and use (their code and method name is actually a duplicate), so no artificial context is needed here.